tests/Makefile

```diff
@@ -201,279 +201,300 @@ test-compile-with-lz4-memory-usage:
 	$(MAKE) clean; CFLAGS=-O0 CPPFLAGS=-D'LZ4_MEMORY_USAGE=LZ4_MEMORY_USAGE_MAX' $(MAKE) all
 
 .PHONY: test-lz4-sparse
+# Rules regarding Temporary test files :
+# Each test must use its own unique set of names during execution.
+# Each temporary test file must begin by an FPREFIX.
+# Each FPREFIX must be unique for each test.
```

Comment:
This is a confirmation for minor/corner case.
As for uniqueness, we also do not accept common prefix of `FPREFIX`.

For example

```
# NG
test-1: FPREFIX = tmp-test-1
	...
	@$(RM) $(FPREFIX)*  # This command (`rm tmp-test-1*`) also removes `tmp-test-1-1*`
...
test-1-1: FPREFIX = tmp-test-1-1
	...
	@$(RM) $(FPREFIX)*
...
```